src/lib.rs

```diff
@@ -95,27 +105,50 @@ enum ConfigAction {
     EXIT,
 }
 
+impl std::fmt::Display for ConfigAction {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        std::fmt::Debug::fmt(self, f)
+    }
+}
+
 pub struct ConfigEvent {
     action: ConfigAction,
     config_ids: Vec<String>,
     config_str: String,
     stage_id: u32,
 }
 
+pub fn setup_logging() {
+    let timer = tracing_subscriber::fmt::time::time();
+    let collector = tracing_subscriber::fmt()
+        .with_max_level(Level::DEBUG)
+        // disable color to make CLion happy
+        .with_ansi(false)
+        .with_thread_ids(true)
+        .with_timer(timer)
+        .finish();
```

Comment:
Here are the settings for logging library, currently, debug level log is enabled.

Reply:
Can we make log level configurable, like vector's : "std::env::var("VECTOR_LOG")", so that for debug/release version, we can collect logs for different levels, no need to manually change the code.